cli/cmd/volume.go

```diff
@@ -77,7 +82,27 @@ var volumeCreateCmd = &cobra.Command{
 		validateNArgs(cmd, 2, 0)
 		volname := cmd.Flags().Args()[0]
 		bricks := cmd.Flags().Args()[1:]
-		fmt.Println("CREATE:", volname, bricks)
+		vol, err := client.VolumeCreate(api.VolCreateReq{
+			Name: volname,
+			Bricks: bricks,
+			Replica: flagCreateCmdReplicaCount,
+			Force: flagCreateCmdForce,
+		})
+		if err != nil {
+			log.WithField("volume", volname).Println("volume creation failed")
+			failure(fmt.Sprintf("Volume creation failed with %s", err.Error()), 1)
+		}
+		fmt.Println("Volume created successfully\n")
```

Comment:
Why not use the tabulator here to get pretty output?

Also, the `volume create` command doesn't show this much information in the old cli.

Reply:
ID is required in the output since user should know the ID of volume created, showing volinfo will be like added value I feel. Previous patchset had tabulator, but I requested this format to match old format.

Reply:
I'd say just the ID would be enough extra information. But we can update the output format later.

Reply:
Updating the patch. Adding the volume info in the volumeInfoCmd().
thanks.